src/main/java/eu/darkbot/api/managers/StatsManager.java

```diff
@@ -0,0 +1,22 @@
+package eu.darkbot.api.managers;
+
+public interface StatsManager {
+
+    int getLevel();
+    long getRuntime();
```

Comment:
```suggestion
    long getRunningTime();
```
`Runtime` sounds like a piece of software, not like a time measurement